django_pdfkit/views.py

```diff
@@ -27,14 +27,17 @@ class PDFView(TemplateView):
 
     #: Set pdfkit options dict.
     pdfkit_options = None
+    
+    def can_render_html(self):
+        return 'html' in request.GET
 
     def get(self, request, *args, **kwargs):
         """
         Return a HTTPResponse either of a PDF file or HTML.
 
         :rtype: HttpResponse
         """
-        if 'html' in request.GET:
+        if self.can_render_html():
```

Comment:
My preference would be to have a class property, for instance `html` which defaults to `True`. That way someone can override it when they define their `urls.py`. For instance;

```python
urlpatterns = [
    url('download', PDFView.as_view(html=False)),
]
```

So for instance the `PDFView` would look something like;

```python
class PDFView(TemplateView):
    ...
    ...

    #: Set to default the PDF display to inline.
    inline = False

    #: Set False to disable HTML rendering
    html = True

    ...
    ...  
```

Can you please also add docs to https://github.com/alexhayes/django-pdfkit/blob/develop/docs/usage.rst#properties in your PR and then I'd be happy to merge.